client/src/Styles/Reviews.css

```diff
@@ -0,0 +1,344 @@
+.reviewContainer {
```

Comment:
why do you have separate css files for App and Reviews?

Reply:
That's what css modules are.